deploy/charts/grafana-multi-tenant-operator/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: grafana-multi-tenant-operator
 description: A Helm chart for the Grafana Multi Tenant Operator by K8Spin.
 type: application
-version: 1.1.0
+version: 1.1.1
```

Comment:
Do not modify this version number. I'll do when triggering a new release :)

Reply:
Alright! I will let you modify this then :)

Reply:
done